app/service/data_svc.py

```diff
@@ -2,9 +2,13 @@
 from base64 import b64encode
 from collections import defaultdict
 from datetime import datetime
-
+from uuid import UUID, uuid4
+import logging
 import yaml
 
+logger = logging.getLogger('DataService')
```

Comment:
let's drop the logger into the __init__ of the DataService to tie it to the class instance (like the OperationService class). don't specify a level though -- it should automatically apply the level that you determine in the local.yml